livekit/src/room/participant/local_participant.rs

```diff
@@ -544,4 +657,194 @@ impl LocalParticipant {
     pub fn kind(&self) -> ParticipantKind {
         self.inner.info.read().kind
     }
+
+    pub async fn perform_rpc(
+        &self,
+        destination_identity: String,
+        method: String,
+        payload: String,
+        response_timeout_ms: Option<u32>,
```

Comment:
since this is the Rust API, I think it would be better to have a RPCData struct? Or maybe a RPCOptions?

Reply:
yeah makes sense - i just went ahead and cipied how sipdtmf, chat, and transcription were doing it